renovate.json

```diff
@@ -1,10 +1,19 @@
 {
-  "automerge": false,
+  "automerge": true,
+  "major": {
+    "automerge": false
+  },
   "branchPrefix": "renovate/",
   "extends": [
-    "config:base"
+    "config:base",
+    ":maintainLockFilesWeekly"
   ],
-  "rangeStrategy": "replace",
+  "masterIssue": true,
   "prConcurrentLimit": 2,
-  "schedule": "every weekend"
+  "prCreation": "not-pending",
```

Comment:
I need to review our circleci config to verify it runs on branches, but this would mean a PR wouldn't show up until the tests pass (meaning, we wouldn't get notified about a PR and then have to wait some amount of time).  Docs @ https://docs.renovatebot.com/configuration-options/#prcreation

Reply:
it doesn't. we changed it to PRs-only to reduce our credit usage

Reply:
aye.  thought that might be the case.  I wonder if you can have it run based on a regex of the branch name...

Reply:
I'll remove this for now so we can get things going.  Thanks